src/vivarium_gates_nutrition_optimization_child/data/loader.py

```diff
@@ -244,26 +245,55 @@ def load_theoretical_minimum_risk_life_expectancy(key: str, location: str) -> pd
 def load_standard_data(key: str, location: str) -> pd.DataFrame:
     key = EntityKey(key)
     entity = utilities.get_entity(key)
-    return interface.get_measure(entity, key.measure, location).droplevel("location")
-
 
-def load_standard_gbd_2019_data_as_gbd_2021_data(key: str, location: str) -> pd.DataFrame:
-    """Read in GBD 2019 data with GBD 2021 ages and years. Zero out neonatal age group data if necessary."""
-    gbd_2019_data = load_standard_data(key, location)
-    data = utilities.reshape_gbd_2019_data_as_gbd_2021_data(gbd_2019_data)
+    use_2019_data_keys = [
+        data_keys.MEASLES.PREVALENCE,
+        data_keys.MEASLES.INCIDENCE_RATE,
+        data_keys.MEASLES.DISABILITY_WEIGHT,
+        data_keys.MEASLES.EMR,
+        data_keys.MEASLES.CSMR,
+        data_keys.LRI.CSMR,
+    ]
 
     neonatal_deleted_keys = [
         data_keys.DIARRHEA.INCIDENCE_RATE,
         data_keys.DIARRHEA.DISABILITY_WEIGHT,
-        data_keys.LRI.INCIDENCE_RATE,
-        data_keys.LRI.DISABILITY_WEIGHT,
         data_keys.MALARIA.INCIDENCE_RATE,
         data_keys.MALARIA.DISABILITY_WEIGHT,
     ]
 
-    if key in neonatal_deleted_keys:
+    both_2019_and_neonatal_deleted = [
+        data_keys.LRI.INCIDENCE_RATE,
+        data_keys.LRI.DISABILITY_WEIGHT,
+    ]
+
+    no_age = [
+        data_keys.POPULATION.CRUDE_BIRTH_RATE,
+    ]
+
+    if key in use_2019_data_keys:
+        data = interface.get_measure(entity, key.measure, location, True).droplevel(
+            "location"
+        )
+        data = data.query("year_start == 2019")
+
+    elif key in neonatal_deleted_keys:
+        data = interface.get_measure(entity, key.measure, location).droplevel("location")
+        data.loc[data.reset_index()["age_start"] < metadata.NEONATAL_END_AGE, :] = 0
+
+    elif key in both_2019_and_neonatal_deleted:
+        data = interface.get_measure(entity, key.measure, location, True).droplevel(
+            "location"
+        )
+        data = data.query("year_start == 2019")
         data.loc[data.reset_index()["age_start"] < metadata.NEONATAL_END_AGE, :] = 0
 
+    else:
+        data = interface.get_measure(entity, key.measure, location).droplevel("location")
+
+    if key not in no_age:
+        data = data.query("age_start < 5")
+
     return data
```

Comment:
If you are always wanting to drop location I would suggest just having return` data.droplevel("location")`since you do that in each branch of this if/else block. Im not confident this will allow you to remove all the different times you have to add that throughout this file but it makes this function a bit cleaner.

Reply:
Makes sense - moved to the return statement